new_dawn_server/questions/api/resources.py

```diff
@@ -0,0 +1,21 @@
+from new_dawn_server.questions.models import AnswerQuestion, Question
+from new_dawn_server.users.api.resources import UserResource
+from tastypie import fields
+from tastypie.resources import ModelResource
+
+
+class QuestionResource(ModelResource):
+    class Meta:
+        allowed_methods = ["get"]
+        queryset = Question.objects.all()
+        resource_name = "question"
+
+
+class AnswerQuestionResource(ModelResource):
+    question = fields.ForeignKey(QuestionResource, "question", related_name="answer_question", full=True)
+    user = fields.ToOneField(UserResource, "user", related_name="answer_question", full=True)
+
+    class Meta:
```

Comment:
Should allow both GET (get answer) and POST (answer question)
Also try adding some tests for GET and POST.
The POST test should be similar to my User register test.
For GET test, it should just involve getting the data and do some validation.
Try figure out it with tastypie test doc.